src/plone/app/mosaic/transform.py

```diff
@@ -152,10 +152,18 @@ def transformIterable(self, result, encoding):
             ])
 
         # Get contentLayout body class
-        if 'template-layout' in body_classes:
+        if (
+            'template-layout' in body_classes or
+            'template-layout_view' in body_classes
+        ):
             adapted = ILayoutAware(context, None)
             if adapted is not None:
-                layout = getattr(adapted, 'contentLayout', None)
+                layout = None
+                if getattr(adapted, 'content_layout_path', False):
+                    # plone.app.blocks > 4.0.0rc1
+                    layout = adapted.content_layout_path()
+                else:
+                    layout = getattr(adapted, 'contentLayout', None)
```

Comment:
This branch might just be removed when https://github.com/plone/plone.app.blocks/pull/47 is eventually merged and released.